.github/workflows/flatpak.yaml

```diff
@@ -0,0 +1,30 @@
+on: 
+  push:
+    tags:
+      - '*'
+workflow_dispatch
+
+jobs: 
+  flatpak:
+     
+    runs-on: ubuntu-20.04
+    
+    steps:
+    - uses: actions/checkout@v2 
```

Comment:
Actually, this action is not needed, as `flatpack-builder` will clone the repo for us later

Reply:
It is needed, to get the flatpak yaml config.